recipe/meta.yaml

```diff
@@ -45,4 +45,5 @@ extra:
     - parthea
     - jreback
     - tswast
-    - jonparrott
+    - theacodes
+    - cpcloud
```

Comment:
@theacodes @cpcloud - are you OK with being added as maintainers?

Reply:
Yep.

Reply:
I just realized Thea is OOO until May. We can send a separate PR to add her once she's back.